.appveyor.yml

```diff
@@ -40,11 +40,13 @@ before_build:
 - cmake -H. -DCMAKE_INSTALL_PREFIX=install -Bbuild_serial -DMFEM_USE_MPI=FALSE
 
 build_script:
-- cmake --build build_parallel
-- cmake --build build_serial
-- cmake --build build_serial --target exec
+- cmake --build build_parallel --config Release -j 4
+- cmake --build build_serial --config Release -j 4
+- cmake --build build_serial --target exec --config Release -j 4
 
 after_build:
 # - cmake --build build_parallel --target check
-- cmake --build build_serial --target RUN_TESTS
+- cmake --build build_serial --target RUN_TESTS --config Release
+- cd build_serial
+- ctest -C Release -R unit --output-on-failure
```

Comment:
I think we can run all tests with `--output-on-failure`: remove the tests runs with `cmake ... --target RUN_TESTS ...` and remove the `-R unit` option from the tests run with `ctest ... --output-on-failure`.

Reply:
`done`